### DISCUSSION TITLE
Request: Reverse Scroll only on Mouse

### DISCUSSION THREAD

**Discussion by gilbigdog** [rxhanson/Multitouch-Community#105] [Ideas]
I use Mouse for half day of work and trackpad for another half. I can reverse the scroll for both Mouse and Trackpad in Systsem Preference but I don't like it for Trackpad. so I install 3rd party application to do it only for Mouse.

It would be good to have in MultiTouch.

**Comment by rxhanson** [comment]
I would like to add this in, it's mainly a matter of prioritizing it over other items. If anyone else wants this, upvote this discussion so I can prioritize it accordingly.
